packages/color-block/src/components/cards/CardsItem.tsx

```diff
@@ -0,0 +1,183 @@
+    useCopy,
+} from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+
+import { ColorName } from '../ColorName';
+import { ColorPickerFlyout } from '../ColorPickerFlyout';
+import { TooltipContent } from '../TooltipContent';
+import { mapColorSpaces } from '../../helpers/mapColorSpaces';
+import { ColorBlockType, ColorSpaceInputValues, ItemProps } from '../../types';
+
+export const CardsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDelete }: ItemProps) => {
+    const { copy, status } = useCopy();
+
+    const mappedFirstColorSpace = mapColorSpaces(colorSpaces[0], color);
+
+    const [colorSpaceInputValues, setColorSpaceInputValues] = useState<ColorSpaceInputValues>({
+        cmykCoated: color.cmykCoated ?? '',
+        cmykNewspaper: color.cmykNewspaper ?? '',
+        cmykUncoated: color.cmykUncoated ?? '',
+        hks: color.hks ?? '',
+        lab: color.lab ?? '',
+        ncs: color.ncs ?? '',
+        oracal: color.oracal ?? '',
+        pantoneCoated: color.pantoneCoated ?? '',
+        pantoneCp: color.pantoneCp ?? '',
+        pantonePlastics: color.pantonePlastics ?? '',
+        pantoneTextile: color.pantoneTextile ?? '',
+        pantoneUncoated: color.pantoneUncoated ?? '',
+        pantone: color.pantone ?? '',
+        ral: color.ral ?? '',
+        threeM: color.threeM ?? '',
+        variable: color.nameCss ?? '',
+    });
+
+    const handleColorSpaceValueChange = (event: FormEvent<HTMLInputElement>) => {
+        const { name, value } = event.currentTarget;
+
+        setColorSpaceInputValues((previousState) => ({ ...previousState, [name]: value }));
+    };
+
+    return (
+        <div
+            data-test-id="cards-item"
+            className="tw-group tw-relative tw-flex tw-flex-col tw-overflow-hidden tw-rounded tw-shadow-inner-line tw-transition-all hover:tw-shadow-inner-line-strong"
+        >
+            {!isEditing ? (
+                <Tooltip
+                    withArrow
+                    position={TooltipPosition.Right}
+                    hoverDelay={0}
+                    content={<TooltipContent colorValue={mappedFirstColorSpace.value ?? ''} status={status} />}
+                    triggerElement={
+                        <div className="tw-overflow-hidden tw-rounded-t tw-bg-[url('https://cdn.frontify.com/img/transparent.png')] tw-bg-[length:10px_10px]">
+                            <div
+                                data-test-id="color-tooltip-trigger"
+                                className="tw-w-full tw-h-[60px] tw-cursor-pointer tw-shadow-inner-line tw-transition-all group-hover:tw-shadow-inner-line-strong"
+                                style={{
+                                    backgroundColor: `rgba(${color.red}, ${color.green}, ${color.blue}, ${
+                                        (color.alpha && color.alpha / 255) || 1
+                                    })`,
+                                }}
```

Comment:
Too complex to read, move it out of the component
```suggestion
                                style={{
                                    backgroundColor: `rgba(${color.red}, ${color.green}, ${color.blue}, ${
                                        (color.alpha && color.alpha / 255) || 1
                                    })`,
                                }}
```